.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 before_script:
   - npm install -g npm
   - npm cache clear
-  - npm install -g broccoli-cli
```

Comment:
Hmmm...
Maybe I don't understand the package.json scripts??
gulp is not executable directly in my shell, but npm run build works.